app_grim_reefer/debug.conf

```diff
@@ -5,7 +5,7 @@
 # in the application. See the README for more details.
 
 # compiler
-CONFIG_DEBUG_OPTIMIZATIONS=y
+CONFIG_SPEED_OPTIMIZATIONS=y
```

Comment:
Wouldn't recommend for a debug overlay config. If you **really** think you need the speed optimizations, put it in the normal prj.conf. Using speed optimizations will just screw up the debugging experience. O2 optimization should be fine though